docs/guidance/map/connecting_publications.md

```diff
@@ -0,0 +1,429 @@
+```{workedexample} Connecting releases from different publications
+:tags: release
+```
+
+# Connecting releases from different publications
+
+Different OCDS publications can contain releases describing the same contracting process. In OCDS, each contracting or planning process is assigned a globally unique identifier, the [ocid](../../schema/identifiers.md#open-contracting-process-identifier-ocid). Ideally, releases describing the same contracting or planning process in different publications would reuse the same ocid so that users can [merge releases](../../schema/merging.md).
```

Comment:
I think this style rule is out of date:

> “ocid”, not “OCID”. Although abbreviations in prose are uppercase, the mixing of upper- and lowercase in the documentation (when referring to the concept versus the field) may cause confusion.

In the Primer we went with OCID. When referring to the field, we use backticks. When we're discussing the concept, we use the acronym. I think this is a solution that reads better.

I think similar changes can be made in this PR.

Reply:
We also use "contracting (or planning) process" unless this noun phrase appears in parentheses (because double parentheses are hard to read).

Reply:
Upper case OCID sounds good. Done in e4312363c6d94793a232f4bb7af46f18f03c04cb

Second change made in 766a6373542387dfb01b5863232ef1e7757ba52d